File: Assinment30&31/program10.java
//write a program which accept one number from user and on its first 4 bits return modified number.

import java.lang.*;
import java.util.*;
//  0000  0000   0000  0000  0000  0000  0000  1000
//  0       0      0    0      0    0      0   8

class Bitwise
{
    public int onBit(int iNo)
    {
        int iMask =0X00000008;  //8
        int iResult = 0;

        iResult = iNo | iMask;
        return iResult;

    }
}

class program10
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number");
        int iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();
        
        int iRet = bobj.onBit(iValue);

        System.out.println("Update Number is : "+iRet);


    }
}

File: Assignment3638/program10.c
//writ a recursive program which accept number from user and return its product of digits
//Input:523
//OutPut : 30

#include<stdio.h>

int Multi(int iNo)
{
 int iDigit = 1;
  static int multi = 1;

  if(iNo != 0)
  {
    
    iDigit = iNo % 10;
    multi =  multi * iDigit;
    iNo = iNo/10;

      Multi(iNo);
  }
  return multi;
}



int main()
{
  int iValue = 0,iRet = 0;

  printf("Enter the Number\n");
  scanf("%d",&iValue);

  iRet = Multi(iValue);

  printf("product is : %d\n",iRet);

  return 0;
}